WalletWasabi/WabiSabi/Client/CoinJoin/Client/CoinJoinClient.cs

```diff
@@ -44,6 +44,8 @@ public class CoinJoinClient
 		string coordinatorIdentifier,
 		CoinJoinCoinSelector coinJoinCoinSelector,
 		LiquidityClueProvider liquidityClueProvider,
+		decimal maxAllowedCoordinationFeeRate,
+		FeeRate maxAllowedCoinjoinMiningFeeRate,
 		TimeSpan feeRateMedianTimeFrame = default,
 		TimeSpan doNotRegisterInLastMinuteTimeLimit = default,
 		CoinjoinSkipFactors? skipFactors = null)
```

Comment:
These are coinjoin configurations. I would suggest to move them all to their own class. The same for `coordinatorIdentifier`.

Reply:
Thanks for implementing this